docs/install/Fedora.md

```diff
@@ -0,0 +1,23 @@
+#INSTALL
+atomic can be installed through below methods
+
+##YUM/DNF
+atomic RPM is available on Fedora 21 and later. So a simple `yum/dnf install atomic` will install it fine.
+
+##Make
+On Fedora 21, You need to install required build dependencies 
+```
+yum install pylint python-docker-py go-md2man
```

Comment:
Why not recommend `yum-builddep atomic`?  That's what I do to get a starting point.